### ISSUE TITLE
About the text-image matching modal

### ISSUE THREAD

**Issue by Pefect96** [CuthbertCai/Ask-Confirm#4]
Hi, How are S-SCAN and T-CMPL models selected for AC training? I see in the test script directly given is 49.pkl? Shouldn't we choose the model that works best in the validation set? If so, how is the best model here measured? With minimum val_losses?

**Comment by CuthbertCai** [comment]
To ensure a fair comparison, both T-CMPL and S-SCAN are trained for 50 epochs, and we choose the final ckpt. We didn't ensure that the best model was selected, because we just want to verify how AC affect different models.

**Comment by Pefect96** [comment]
OK. So the object Detector also the last model of epoch? Why does AC not use validation sets when training? How do you know if it converges? Because every time I train the model, the result is different.

**Comment by CuthbertCai** [comment]
The object detector adopts ckpt from [bottom-up top-down attention][1]. For AC training, I directly choose a certain epoch to stop the training (for both S-SCAN and T-CMPL). 
According to my experiments, if you stop the training at the same epoch every time, the results seem very similar.

[1]:https://github.com/peteanderson80/bottom-up-attention
